README.rst

```diff
@@ -92,3 +92,76 @@ By default SQLite memory database will be used.
 .. code-block:: shell
 
     py.test test --test-db-url=sqlite:///test_db.sql
+    py.test test --test-db-url=mysql+mysqlconnector://root:password@localhost:3306/nameko_sqlalchemy_test
+
+Helper functions
+----------------
+
+run_query
+^^^^^^^^^
+This function provides a way to protect against losing uncommitted changes when a database connection error occur.
+
+Usage
+"""""
+
+.. code-block:: python
+
+    from sqlalchemy import create_engine
+    from sqlalchemy.orm import sessionmaker
+
+    from nameko_sqlalchemy import run_query
+
+
+    engine = create_engine('postgresql://username:password@localhost/test')
+    Session = sessionmaker(bind=engine)
+    db_session = Session()
+
+
+    def get_example_data():
+        return db_session.query(ExampleModel).all()
+
+    example_data = run_query(db_session, query)
```

Comment:
where does the query come from?

Reply:
Good point, it should be `get_example_data`, I will fix this.